router/batchrouter/batchrouter.go

```diff
@@ -2434,3 +2436,25 @@ func (brt *HandleT) updateProcessedEventsMetrics(statusList []*jobsdb.JobStatusT
 		}
 	}
 }
+
+func tryUnmarshalJSON(jobID int64, data []byte, v interface{}) (err error) {
+	c := cap(data)
+	l := len(data)
+
+	startingData := make([]byte, l)
+	copy(startingData, data)
```

Comment:
Does this add any significant overhead?

Reply:
Not particularly concerned about the overhead but not sure how useful it can be as a metric given that the race could happen exactly when we copy rather than we unmarshal. I'll get rid of it. The plan is to get the configuration used to trigger this and the jobID to further investigate, I don't think the copy will solve much. @lvrach any thoughts since you added this?

Reply:
Please keep in mind that it is possible for the job to get lost pretty quickly from the table, without leaving other trace, since AFAIK we don't keep backups of all jobs in hosted environment

Reply:
> race could happen exactly when we copy rather than we unmarshal
Do you think that this going to cause a panic? Well, the defer will deal with that I believe. if we manage to capture `startingData` != `data` it would be very useful.